### PR TITLE
Fix minor issues in the fsm design doc

### PR DESCRIPTION
**Pull request by vishnoianil**

### CODE REVIEW

File: docs/development/design/fsm.md [hunk 1]
```diff
@@ -48,11 +47,12 @@ nexd may be in one of these states:
 
 1. UNAUTHENTICATED: No configuration has been provided
 1. AUTHENTICATING: Performing authentication, either interactively or via token
-1. AUTHENTICATED: User is succesfully authenticated
+1. AUTHENTICATED: User is successfully authenticated
 1. UNREGISTERED: Authenticated but device has not been registered with the control plane
 1. REGISTERED: Device is registered with the control plane, but data plane is not established
 1. PEERING: Device is connected to control plane and fetching peer listings
 1. UP: Data plane is established
+1. DOWN: Data plane is not established
```
Comment (russellb) [inline docs/development/design/fsm.md:55]:
```suggestion
1. DOWN: Data plane has been administratively turned off while retaining a connection to the control plane.
```

File: docs/development/design/fsm.md [hunk 2]
```diff
@@ -28,7 +28,6 @@ stateDiagram-v2
     REGISTERED --> UNAUTHENTICATED: "logout"
     UNREGISTERED --> REGISTERING: "register_device"
     UNREGISTERED --> UNAUTHENTICATED: "logout"
-    REGISTERED --> UNAUTHENTICATED: "logout"
```
Comment (russellb) [inline docs/development/design/fsm.md:31]: Why is this line removed?

**Review by russellb — CHANGES_REQUESTED** [review]

File: docs/development/design/fsm.md (hunk 1 above)
Comment (vishnoianil) [reply docs/development/design/fsm.md:55]: @russellb looking at the current fsm graph, transition to DOWN Is not always administratively down, "peering_error" can transition it to DOWN as well.

File: docs/development/design/fsm.md (hunk 2 above)
Comment (vishnoianil) [reply docs/development/design/fsm.md:31]: duplicate: see line 28

Comment (vishnoianil) [reply docs/development/design/fsm.md:31]: generated graph, shows duplicate lines from REGISTERED-> UNAUTHENTICATED (https://github.com/dave-tucker/apex/blob/b39acf40d7c51cdfdb50db6759aa1cf09151b36c/docs/development/design/fsm.md)

File: docs/development/design/fsm.md (hunk 1 above)
Comment (russellb) [reply docs/development/design/fsm.md:55]: Ah, you're right.

```suggestion
1. DOWN: Data plane is not established, either due to an error or it has been placed down administratively using nexctl
```

**Review by russellb — APPROVED** [review]
